app/models/image.rb

```diff
@@ -11,6 +11,7 @@ class Image < ActiveFedora::Base
   include DogBiscuits::Geo
   include DogBiscuits::PartOf
   include DogBiscuits::PlaceOfPublication
+  include IiifPrint.model_configuration
```

Comment:
the GenericWork one specifies a pdf_split model. Does this one need to?

Reply:
Set up as is, this will not split PDFs but it will set it to go through the default derivatives (JP2, Tiff, and Text Extraction)

ref: https://playbook-staging.notch8.com/en/DEMOS/IiifPrint/i14-toggle-derivatives